pkg/mkimage-iso-efi/make-efi

```diff
@@ -14,16 +14,14 @@ copy() {
 }
 
 #
-# Extract partitions from stdin or look them up in /bits
+# Extract partitions from stdin to a working directory
+# We could do this in /tmp, but it might be really big, and for cases where
+# /tmp is tmpfs, that might use up a lot of memory
 #
-mkdir -p /parts 2>/dev/null || :
-cd /parts
-if [ -d /bits ]; then
-   ln -s /bits/* .
-   rm \* >/dev/null 2>&1 || :
-else
-   bsdtar xzf -
-fi
+TMPDIR=/var/efiparts-$$
```

Comment:
Not critical at all, but another option would be:
```bash
TMPDIR=$(mktemp -d /var/efiparts-XXXXXX)
trap 'rm -rf "$TMPDIR"' EXIT
```

Reply:
Be critical! I can handle it. 

I thought about it. I was concerned that these images are so big, if /tmp is tmpfs, it might eat up too much memory.